arch/xtensa/src/esp32/esp32_rt_timer.c

```diff
@@ -60,17 +61,27 @@
 #define ESP32_RT_TIMER            0 /* Timer 0 */
 
 /****************************************************************************
- * Private Data
+ * Private Types
  ****************************************************************************/
 
-static int s_pid;
+struct esp32_rt_priv_s
+{
+  int pid;
+
```

Comment:
```suggestion
```

Reply:
These spaces were added deliberately to separate members of different purpose.